src/app/themes/Theme.ts

```diff
@@ -26,6 +26,12 @@ export const theme = createTheme({
     },
   },
   overrides: {
+    MuiLink: {
+      root: {
+        color: '#35aed7',
```

Comment:
Can we use colors in contants here?

Reply:
I tried a lot but it wasn't working. I'll try again because maybe it was cache or something like that